bonnie-rest/src/main/java/com/cgi/hexagon/bonnierest/OrderController.java

```diff
@@ -49,6 +49,17 @@ public ResponseEntity<List<Order>> getAllOrders() {
         }
     }
 
+    @GetMapping("/find/{status}")
+    public ResponseEntity<List<Order>> findByStatus(@PathVariable Status status) {
```

Comment:
We want to provide the users an API that let's them list the orders which is not yet claimed. So I think an endpoint that searches for only unclaimed orders would fit better. Something like find/unclaimed or find/unassigned. I'm not really sure that we would need to provide an endpoint for all types of statuses